src/timer.c

```diff
@@ -44,9 +44,6 @@ static int timer_less_than(const struct heap_node* ha,
    */
   if (a->start_id < b->start_id)
     return 1;
-  if (b->start_id < a->start_id)
-    return 0;
-
   return 0;
 }
```

Comment:
I'd be okay with replacing this with `return a->start_id < b->start_id;` but either way is fine.

Reply:
Done this as well 👍